website/pages/docs/[[...page]].jsx

```diff
@@ -12,7 +12,7 @@ function DocsLayout(props) {
   return (
     <DocsPage
       productName={productName}
-      productSlug="red"
+      productSlug="boundary"
```

Comment:
Actually, keep this as `red` for now!   This was our product codename prior to launch, and styles are still dependent on this.

We'll make this change in the near future, but first we'll have to do some component upgrades.

```suggestion
      productSlug="red"
```

Reply:
Fair, I wasn't aware this affects styles too.

I suppose you know already, but in case it wasn't clear from the PR - it breaks the `Edit this page` URL on all pages.